### PR TITLE
chore: fix-ci-samples

### PR DESCRIPTION
**Pull request by ryanbas21**
https://pingidentity.atlassian.net/browse/SDKS-3516

### CODE REVIEW

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -62,7 +69,7 @@ jobs:
         with:
           name: Playwright Results - ${{ runner.os }} - ${{ matrix.node }} - ${{ github.run_attempt }}
           path: |
-            javascript/reactjs-todo/test-results
-            javascript/reactjs-todo/playwright-report
-            javascript/angular-todo/test-results
-            javascript/angular-todo/playwright-report
+            ./javascrptit/reactjs-todo/test-results
```
Comment (george-bafaloukas-forgerock) [inline .github/workflows/CI.yml:72]: `javascrptit` typo

File: javascript/angular-todo/src/environments/environment.ts [hunk 2]
```diff
@@ -1,10 +1,10 @@
 export const environment = {
-   AM_URL: 'undefined',
-   REALM_PATH: 'undefined',
-   WEB_OAUTH_CLIENT: 'undefined',
-   JOURNEY_LOGIN: 'undefined',
-   JOURNEY_REGISTER: 'undefined',
-   API_URL: 'undefined',
+   SERVER_URL: 'https://openam-sdks.forgeblocks.com/am',
+   REALM_PATH: 'alpha',
+   WEB_OAUTH_CLIENT: 'CentralLoginOAuthClient-',
```
Comment (george-bafaloukas-forgerock) [inline javascript/angular-todo/src/environments/environment.ts:4]: Does the client have a `-` at the end or is that a mistake?

**Review by george-bafaloukas-forgerock — COMMENTED** [review]
@ryanbas21 Looks good apart from the typo

File: javascript/angular-todo/src/environments/environment.ts (hunk 2 above)
Comment (ryanbas21) [reply javascript/angular-todo/src/environments/environment.ts:4]: It does

**Pull request merged**